x/chainlink/client/rest/queryFeed.go

```diff
@@ -11,31 +10,18 @@ import (
 	"github.com/gorilla/mux"
 )
 
-func listFeedDataHandler(clientCtx client.Context) http.HandlerFunc {
+func listFeedDataByFeedIdHandler(clientCtx client.Context) http.HandlerFunc {
 	return func(w http.ResponseWriter, r *http.Request) {
 		vars := mux.Vars(r)
 		feedId := vars["feedId"]
 
-		fmt.Println("???????", feedId)
-
-		params := &types.QueryFeedDataRequest{
-			FeedId:     feedId,
-			Pagination: nil,
-		}
-
-		queryClient := types.NewQueryClient(clientCtx)
-		res, err := queryClient.FeedDataByID(context.Background(), params)
+		res, height, err := clientCtx.QueryWithData(fmt.Sprintf("custom/%s/%s/%s", types.StoreKey, types.QueryFeedData, feedId), nil)
```

Comment:
this will eventually include the `roundid` too, right?

Reply:
yes, we need to impl roundID first and it will be param in query later

Reply:
idk, this doesnt need to be changed right now, but eventually i think

Reply:
correct, that is the path after we have roundID